Inc/ReadGps.h

```diff
@@ -1,3 +1,48 @@
 #pragma once
 
-void readGpsTask(void const* arg);
\ No newline at end of file
+void readGpsTask(void const* arg);
+UART_HandleTypeDef huart1;
+
+static uint8_t CFG_MSG1[] = {0x06, 0x01, 0x08, 0x00, 0x01, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
```

Comment:
let's name these starting with `GPS` like GPS_CFG_MSG. I think this would also be easier to work with, if it was a 2D array. Also they shoudl be const.